docs/content/v1/usage/custom-directive.md

```diff
@@ -19,14 +21,76 @@ A `:` custom directive will interpret in JS whatever you pass as value.
 </script>
 ```
 
-outputs
+Outputs:
 
 ```html
 <a href="/user/144">Visit Profile</a>
 ```
 
-[Boolean attributes](https://www.w3.org/TR/html5/infrastructure.html#sec-boolean-attributes)
+## A Utility for Readability
+
+This is particularly useful to lighten your `<template>` tag in order to keep it readable.
+
+Instead of:
+
+```html
+<template>
+  <p :if="state.count < 5 && (state.canDoThis || state.attendees.includes('Bob'))">
+    My conditional paragraph
+  </p>
+</template>
+
+<script>
+  export default class extends Lego {
+    state = { count: 6, canDoThis: true, attendees: ['John', 'Mary'] }
+  }
+</script>
+```
+
+You can use getters:
+
+```html
+<template>
+  <p :if="this.canDisplayMessage">
+    My conditional paragraph
+  </p>
+</template>
+
+<script>
+  export default class extends Lego {
+    state = { count: 6, canDoThis: true, attendees: ['John', 'Mary'] }
+
+    get isBobTakingPart() {
+      return this.state.attendees.includes('Bob')
+    }
+
+    get canDisplayMessage() {
+      return this.state.count < 5 && (this.state.canDoThis || this.isBobTakingPart)
+    }
+  }
+</script>
+```
+
+## [Boolean attributes](https://www.w3.org/TR/html5/infrastructure.html#sec-boolean-attributes)
+
+When a boolean value is evaluated to `true`, the attribute in set as empty. While when evaluated to `false`, the attribute is removed from the DOM.
+
+### Example
+
+```html
+<template>
+  <input type="checkbox" :checked="state.agreed" :required="state.mustAgree" />
+</template>
+
+<script>
+  export default class extends Lego {
+    state = { agreed: false, mustAgree: true }
+  }
+</script>
+```
+
+Outputs:
 
-Example: `<input type=checkbox :checked="state.agreed" :required="state.mustAgree">`.
-With the following state: `this.state = { agreed: false, mustAgree: true }` would render
-`<input type=checkbox required="required">`.

```

Comment:
I think the existing documentation was wrong here. In my experience, `petit-dom` evaluates a true boolean to an empty string